base.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+
+  <remote fetch="https://android.googlesource.com" name="aosp"/>
+  <remote fetch="https://github.com/kaiostech/" name="kaios"/>
```

Comment:
Thanks for taking care of this. IMO, these two lines can be placed in aosp.xml and base-q.xml respectively.  Then we can reduce one include file.

Reply:
let me check